src/main/java/io/kubernetes/KabaneroClient.java

```diff
@@ -96,35 +102,41 @@ private static ApiClient getApiClient() throws IOException, GeneralSecurityExcep
         return client;
     }
 
-    public static List<KabaneroInstance> getInstances() throws IOException, ApiException, GeneralSecurityException {
+    public static KabaneroList getInstances() throws IOException, ApiException, GeneralSecurityException {
         ApiClient client = KabaneroClient.getApiClient();
-
         String namespace = "kabanero";
+        try{
+            KabaneroApi api = new KabaneroApi(client);
+            KabaneroList kabaneros = api.listKabaneros(namespace, null, null, null);
+            return kabaneros;
+        }catch (Exception e) {
+            LOGGER.log(Level.WARNING, "Error with fetching kabanero instances - " + e.getStackTrace().toString());
+            return null;
+		}
```

Comment:
fix formatting here and just pass `e` as loggers second param. `LOGGER.log(Level.WARNING, "Error with fetching kabanero instances", e);`